src/test/kotlin/study/domain/RaceCarTest.kt

```diff
@@ -6,7 +6,7 @@ import org.junit.jupiter.api.Test
 class RaceCarTest {
     @Test
     fun move() {
-        val raceCar = RaceCar()
+        val raceCar = RaceCar("name")
```

Comment:
아래와 같이 선언해도 테스트는 정상 실행될것 같은데, 요구사항을 만족할수 있도록 정의해보세요 ! 
```suggestion
        val raceCar = RaceCar("name12345")
```

Reply:
수정하였습니다~!